docs/faq/BundleSize.md

```diff
@@ -0,0 +1,61 @@
+# How to reduce  `redux-form`'s bundle size?
+
+For convenience, Redux Form exposes its full API on the top-level `redux-form` import. 
+However, this causes the entire Redux Form library and its dependencies to be included in client bundles that 
+include code that imports from the top-level import.
+
+Instead, the bindings exported from `redux-form` are also available in `redux-form/lib` and `redux-form/es`. 
+You can import directly from those to avoid pulling in unused modules. 
+
+Example:
+```js
+import { reduxForm, Field, FieldArray } from 'redux-form'
+```
+
+use:
+
+```js
+import reduxForm from 'react-router/es/reduxForm'
+import Field from 'react-router/es/Field'
+import FieldArray from 'react-router/es/FieldArray'
+```
+
+
+Use `es` if you are using a bundler that can process ES modules like `webpack@2` or `rollup`, otherwise use `lib`.
+
+The public API available in this manner is defined as the set of imports available from the top-level `react-router` module.
```

Comment:
Next time I 'inspire' myself on the work of others, do cmd+R. 10 year old me would not have done this kind of mistake. 😃